src/os/bluestore/BlueFS.h

```diff
@@ -776,6 +777,9 @@ class BlueFS {
   }
   uint64_t debug_get_dirty_seq(FileWriter *h);
   bool debug_get_is_dev_dirty(FileWriter *h, uint8_t dev);
+  void debug_interrupt_async_compact(uint32_t stop_at) {
```

Comment:
Let's have uniform API (similar to one proposed at https://github.com/ceph/ceph/pull/60732) for this sort of things (func name is debatable):
set_debug_tracepoint(event_mask, callback)